src/Tests/PathautoLocaleTest.php

```diff
@@ -84,5 +93,92 @@ function testLanguageAliases() {
     // suffix.
     $this->assertEntityAlias($node, '/content/english-node-1', LanguageInterface::LANGCODE_NOT_SPECIFIED);
   }
-}
 
+  /**
+   * Test that patterns work on multilingual content.
+   */
+  function testLanguagePatterns() {
+    $this->drupalLogin($this->rootUser);
+
+    // Add French language.
+    ConfigurableLanguage::createFromLangcode('fr')->save();
+
+    // Enable content translation on articles.
+    \Drupal::service('content_translation.manager')->setEnabled('node', 'article', TRUE);
+    drupal_static_reset();
+    \Drupal::entityManager()->clearCachedDefinitions();
+    \Drupal::service('router.builder')->rebuild();
+    \Drupal::service('entity.definition_update_manager')->applyUpdates();
+
+    // Create a pattern for English articles.
+    $pattern = $this->createPattern('node', '/the-articles/[node:title]');
```

Comment:
We are already doing that at https://github.com/md-systems/pathauto/blob/8.x-1.x/src/Tests/PathautoUiTest.php#L88 but I see that it does not test language and bundle selection. I will adjust this then.